pom.xml

```diff
@@ -29,11 +29,11 @@
     <maven.dependency.plugin.version>3.2.0</maven.dependency.plugin.version>
     <maven.jar.plugin.version>3.2.0</maven.jar.plugin.version>
     <maven.resources.plugin.version>3.1.0</maven.resources.plugin.version>
-    <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
-    <quarkus.plugin.version>1.12.1.Final</quarkus.plugin.version>
+    <maven.surefire.plugin.version>3.0.0-M6</maven.surefire.plugin.version>
+    <quarkus.plugin.version>3.0.0.Alpha4</quarkus.plugin.version>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
-    <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-    <quarkus.platform.version>1.12.1.Final</quarkus.platform.version>
+    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+    <quarkus.platform.version>3.0.0.Alpha4</quarkus.platform.version>
     <testcontainers.version>1.15.2</testcontainers.version>
     <okhttp.version>4.9.3</okhttp.version>
```

Comment:
Don't we have to update or remove this? It should be a dependency inherited from Quarkus? No?

Reply:
Nope.

Reply:
Are we talking about the same line? I meant the okhttp version.

Reply:
No we aren't :grin: ...  I'll check that and see.